QUALITY_DECLARATION.md

```diff
@@ -114,7 +114,9 @@ This includes:
 
 Changes are required to make a best effort to keep or increase coverage before being accepted, but decreases are allowed if properly justified and accepted by reviewers.
 
-Current coverage statistics can be viewed [here](https://ci.ros2.org/job/ci_linux_coverage/85/cobertura/src_ros_class_loader_include_class_loader/) and [here](https://ci.ros2.org/job/ci_linux_coverage/85/cobertura/src_ros_class_loader_include_class_loader/). This package does not yet meet the 95% coverage guideline, but it is currently above 90%.
+This package has testing coverage of at least 95%.
+Current coverage statistics can be viewed [here](https://ci.ros2.org/job/nightly_linux_coverage/lastSuccessfulBuild/cobertura/).
```

Comment:
The link would be this, right?
```suggestion
Current coverage statistics can be viewed [here](https://ci.ros2.org/job/nightly_linux_coverage/lastSuccessfulBuild/cobertura/src_ros_class_loader_src/).
```

Reply:
I've started just linking to the full cobertura report because the statistics for a package has to be composed from multiple lines. In this case, there is coverage in the include and src directories.

I'm open to better approaches for this sort of thing though.

Reply:
Mmm, isn't the full coverage reported under its package folder? The fact that the coverage is tested with more packages is stated in the developer guide, IMO it should be enough.

Reply:
Since there are functions in the include directory, coverage for class_loader should be composed from:

https://ci.ros2.org/job/nightly_linux_coverage/lastSuccessfulBuild/cobertura/src_ros_class_loader_include_class_loader/
and
https://ci.ros2.org/job/nightly_linux_coverage/lastSuccessfulBuild/cobertura/src_ros_class_loader_src/

As described in the developer guide, the coverage statistics should include "src.*.<repository_name>.<package_name>.*" and "build/.<repository_name>.*" (if it exists). Depending on the package that may be a lot of links to include in a single paragraph. Seeing as the number of links might grow or shrink as the package grows or shrinks, I figured it would be a lot easier to just link to the main page.